test/Streamly/Test/Data/Unfold.hs

```diff
@@ -449,6 +450,17 @@ enumerateFromToFractional =
 -- Stream transformation
 -------------------------------------------------------------------------------
 
+scan :: Property
+scan =
+    property
+        $ \f _list ->
+              let _fA = apply f :: Int -> Int
+                  _x = _list :: [Int]
+                  unf = UF.scan Fold.sum UF.fromList
+                  list = [1,2,3,4] :: [Int]
+                  mList = [1,3,6,10] :: [Int]
```

Comment:
Can we use the passed list instead and compare it against the Data.List scan function?